Tests/MoyaTests/TestHelpers.swift

```diff
@@ -230,13 +230,13 @@ extension URL {
     }
 }
 
-// MARK: - Image Test Helpers
-// Necessary since Image(named:) doesn't work correctly in the test bundle
+// MARK: - PlatformImage Test Helpers
+// Necessary since PlatformImage(named:) doesn't work correctly in the test bundle
 extension ImageType {
     class TestClass { }
 
     static var testImage: ImageType {
```

Comment:
Was a little surprised this returns the underlying `ImageType`, but thought I'd leave it as-is.